.github/workflows/renovate.yaml

```
@@ -0,0 +1,35 @@
+name: Renovate
+on:
+  push:
+    paths:
+      - ".github/workflows/renovate.yaml"
+      - ".github/renovate-config.js"
+  schedule:
+    - cron: "0 7 * * 5"
+  workflow_dispatch:
+
+env:
+  ACTIONS_STEP_DEBUG: ${{ secrets.ACTIONS_STEP_DEBUG }}
+
+jobs:
+  renovate:
+    runs-on: [self-hosted, linux, x64]
+
+    permissions:
+      contents: write
+      pull-requests: write
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+
+      - name: Self-hosted Renovate
+        uses: renovatebot/github-action@e084b5ac6fd201023db6dd7743aec023babb02c8 # v41.0.13
+        with:
+          configurationFile: ".github/renovate-config.js"
+          token: "${{ secrets.RENOVATE_TOKEN }}"
```

```suggestion
          token: "${{ secrets.GITHUB_TOKEN }}"
```

This way we leverage the permissions from https://github.com/fluent/fluent-operator/pull/1535/files#diff-f87681f1c7db60655a75838d0610544d4b75e26c6909a97b97f5ec4aa323c7aaR18-R20

As we don't need access to private repositories and such, above token from suggestion should be sufficient. It will be allowed to commit using `contents: write` and to create the PR using `pull-requests: write`.

---

good point, let me try on fork before updating.